src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,127 @@
+                                    amet consectetur adipisicing elit.
+                                </Typography>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                    <Grid item xs={12} md={7} sx={isMobile ? ({ mt: 4 }) : (null)}>
+                        <Box component="form">
+                            <Grid container direction='row'>
+                                <Grid item xs={9} sm={10}>
+                                    <TextField id="outlined-basic" label="Enter your email address" variant="outlined" type="email" sx={{ width: '100%', '& fieldset': { borderRadius: '4px 0px 0px 4px' }, '& .Mui-focused .MuiOutlinedInput-notchedOutline':{borderWidth:'1px'}}} />
+                                </Grid>
+                                <Grid item xs={3} sm={2}>
+                                    <Button variant="contained" sx={{ width: '100%', height: '100%', borderLeft: 0, borderRadius: '0px 4px 4px 0px', boxShadow: 'none', '&:hover': { boxShadow: 'none' } }}>Subscribe</Button>
+                                </Grid>
+                            </Grid>
+                            <Grid container direction='row'>
+                                <Grid item xs={12} sx={{ mt: 2 }}>
+                                    <Typography>
+                                        Subscribe to the Libernetix mailing list to receive latest information on our updates
+                                    </Typography>
+                                </Grid>
+                            </Grid>
+                            <Grid container direction='row' sx={{ mt: 4 }}>
+                                <Grid item xs={6} md={4}>
+                                    <Grid container direction='row'>
+                                        <FooterLinkHeading>Solutions</FooterLinkHeading>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Payment Gateway</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Merchant Dashbord</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Card Acquiring</FooterLink>
+                                    </Grid>
+                                    <Grid container direction='row' sx={{ mt: 1 }}>
+                                        <FooterLink href="#">Analytics &amp; Reporting</FooterLink>
+                                    </Grid>
+                                </Grid>
+                                <Grid item xs={6} md={4}>
+                                    <Grid container direction='row'>
```

Comment:
A lot of repeating code here. Would not it be better to use map funtion for all these links? For example put all these Link names in array and just map throught it. Not sure if it is a best sollution but it looks way better than all these Grids